src/main/java/com/softserveinc/dokazovi/controller/EndPoints.java

```diff
@@ -4,5 +4,6 @@
 
 	public static final String WEB_API = "/api";
```

Comment:
If you will use config from the application.property
"server.servlet.context-path=/api"
you should update other controllers with @RequestMapper(EndPoints.WEB_API)